pkg/hive/health/provider.go

```diff
@@ -67,6 +70,20 @@ func (p *provider) ForModule(mid cell.FullModuleID) cell.Health {
 			if p.stopped.Load() {
 				return fmt.Errorf("provider is stopped, no more updates will take place")
 			}
+
+			if len(s.Message) >= maxHealthProviderMsgLen {
+				p.logger.Error("health status update for exceeded max message length and has been truncated",
```

Comment:
maybe something more concise?
`Health status update truncated due to exceeding maximum message length`

and maybe also include the current length?

```
p.logger.Error("Health status update truncated due to exceeding maximum message length",
					"id", s.ID.String(), "maxLength", maxHealthProviderMsgLen, "len", len(s.Message))
```